Cargo.toml

```diff
@@ -17,6 +17,16 @@ libsecp256k1 = "0.3.5"
 rand = "0.7.3"
 hex = "0.4.2"
 sha2 = "0.8"
+lazy_static = "1.4"
+
+[dependencies.class_group]
+git = "http://github.com/LLFourn/class"
```

Comment:
Can we have a tag/commit dependency here please so we don't need to rely on the lockfile (which is not checked it atm).

Reply:
done.